hack/install-gox.sh

```diff
@@ -24,12 +24,10 @@ cleanup() {
 trap cleanup EXIT
 
 install_gox() {
-  gobin="$(go env GOPATH)/bin"
   cd "${scratch_dir}"
-  env GOPATH="${scratch_dir}" \
```

Comment:
Ah.. now I understand why you originally had the DRY patch in this PR. For the install helpers, I would not necessarily have removed the `GO111MODULE` env var, though.

But since you did it, can you also add a version tag to each installed tool?

Reply:
Sure, it's a better idea.

Reply:
Added versions.